venvs/common.py

```diff
@@ -8,9 +8,25 @@
 import attr
 import click
 import filesystems.native
+import pytoml
+import tomlkit
```

Comment:
Maybe it's just because I haven't used this library yet so I don't appreciate it yet but ISTM it would be simpler and less error-prone to just stick with `pytoml` and just append valid TOML to the file as a string rather than parsing it at all.

Reply:
As of 47395700d027d1b26979446584cedbb772febfde, `pytoml` is being used only `converge.py`. We could go either way here. `pytoml` would be simpler ; `tomlkit` it would make it possible to have commands like `remove` without messing up the user's config.